pom.xml

```diff
@@ -486,6 +722,94 @@
 					-->
 				</configuration>
 			</plugin>
+
+
+			<plugin>
+				<groupId>org.openjfx</groupId>
+				<artifactId>javafx-maven-plugin</artifactId>
+				<version>0.0.5</version>
+				<executions>
+					<execution>
+						<id>default-cli</id>
+						<configuration>
+							<mainClass>org.janelia.saalfeldlab.paintera.Paintera</mainClass>
+							<options>
+								<option>-Xmx50g</option>
```

Comment:
That would be too much for my laptop.  Is this being overridden by the jgo launcher?  Do we need this here or can we live with some default Xmx depending on the platform?

Reply:
Meant to remove this actually, my bad